site/src/docs/getting-started/tutorials/migrate-to-3.0.0.mdx

```diff
@@ -0,0 +1,107 @@
+---
+slug: '/getting-started/tutorials/migrate-to-3.0.0'
+title: 'Migrate to 3.0.0'
+navTitle: 'Migrate to 3.0.0'
+---
+
+import ExternalLink from '../../../components/ExternalLink';
+import LeadParagraph from '../../../components/LeadParagraph';
+import InternalLink from '../../../components/InternalLink';
+
+# Migrate to version 3.0.0
+This guide helps you to start using HDS version 3.0.0. 
+
+## Breaking changes
+This major release contains some breaking changes. Pay attention to these when you update your project to use the new version.
+
+### Navigation replaced with Header
+Old navigation was removed as previously informed. The aim is to provide more flexible navigation solution that supports also larger sites and is consistent with hel.fi-styles.
+This new navigation is not updated automatically, you need to switch to it manually as the name of the component is different.
+
+If you wish to make it as similar to previous navigation, you only need ActionBar-subcomponent.
+
+The navigation was rewritten to fix bugs, make it more modular, provide support for larger sites and unify the styles.
+
+- <InternalLink size="M" href="/components/header">Header component</InternalLink>
+- <InternalLink size="M" href="/patterns/navigation/building-navigation/">Navigation pattern</InternalLink>
+
+### New footer
+Footer has been refactored and the new version is more modular. The new component also is named footer, so check that needed properties are set correctly.
+
+- <InternalLink size="M" href="/components/footer">Footer component</InternalLink>
+
+### Deprecated rectangular tags and statuslabels
+Rectangular tags and status labels have been deprecated and removed. Use rounded version instead.
```

Comment:
This sounds a bit misleading like users would need to do more work than they should, when in reality they don't need to do any changes if they were using default Tag or StatusLabel as they automatically are rounded now. But if they were using the new RoundedTag/RoundedLargeTag or `<StatusLabel variant="rounded" .../>`then those syntaxes will be faulty now. They have been deprecated and removed also. 

I would suggest these changes:
- [ ] Change the heading to "Deprecated rectangular variants to Tags and StatusLabels"
- [ ] Change the description to also cover RoundedTag, LargeRoundedTag and property removal of variant to StatusLabel
- [ ] Change the description to drive it home to the reader, that by default the components have rounded corners now